Provider, Publisher, Invoice Issuer, Datagenerator Consistency Refactor (#834)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/invoiceissuer.md

# Invoice Issuer

An Invoice Issuer is the entity responsible for invoicing and receiving payment from the customer for the consumed [*resources*](#glossary:resource) or [*services*](#glossary:service), including resellers, marketplaces, or billing intermediaries. Commonly used in cost analysis and reporting.

The InvoiceIssuer column adheres to the following requirements:

* The InvoiceIssuer column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
* This column MUST be of type String and MUST NOT contain null values.

> See [Appendix: Origination of cost data](#originationofcostdata) section for examples of Invoice Issuer values that can be used for various purchasing scenarios.

Reviewer: - **Editorial:**  The reference to the appendix (both the title and the URL) needs to be updated
- **Note:** The examples are no longer limited to purchasing scenarios?  

```suggestion
See [Appendix: Entity Identification Examples](#entityidentification) section for examples of Invoice Issuer values for various use case scenarios.
```
